dash/dash.py

```diff
@@ -117,14 +117,17 @@
 _ID_STORE = "_pages_store"
 _ID_DUMMY = "_pages_dummy"
 
-page_container = html.Div(
-    children=[
-        dcc.Location(id=_ID_LOCATION),
-        html.Div(id=_ID_CONTENT),
-        dcc.Store(id=_ID_STORE),
-        html.Div(id=_ID_DUMMY),
-    ]
-)
+if not hasattr(html, "Div"):
```

Comment:
It also needs to check `dcc` because that is also used.

Reply:
I changed it to a try/except AttributeError.  If it's better to check for all the attributes with hasattr, please let me know and I'll change it back

Reply:
LGTM - I might just suggest adding a comment about _why_ we need this, so we don't accidentally remove it later.